compiler-rt/lib/scudo/standalone/combined.h

```diff
@@ -1570,7 +1570,7 @@ class Allocator {
     if (TabSize > UINT32_MAX / kFramesPerStack)
       return;
     u32 RingSize = static_cast<u32>(TabSize * kFramesPerStack);
-    DCHECK(isPowerOfTwo(RingSize));
+    CHECK(isPowerOfTwo(RingSize));
 
```

Comment:
I thought you would like to simply return false instead of abortion in #82683, why do we want to change to CHECK ?

Reply:
because this is impossible by construction, and this just makes it explicit. a power of two times a power of two is always a power of two

Reply:
I think we can remove the DCHECK. It has been verified in the StackDepot::init() and I think it makes more sense to check it there because it uses the value.